docs/userGuide/syntax/textStyles.mbdf

```diff
@@ -26,7 +26,7 @@ Syntax added by MarkBind:
 <include src="codeAndOutput.md" boilerplate >
 <variable name="highlightStyle">markdown</variable>
 <variable name="code">
-****Super Bold****, ++Underline++, $$Underline$$, ==Highlight==, %%Dim%%, Super^script^, Sub~script~
```

Comment:
lgtm 👍 

just one nit here: let's remove the other two in the docs, or mark that they are deprecated

Reply:
Have made the changes! I decided to remove them since we want users to use the latest syntax, and it doesn't seem  easy to mark them as deprecated given the format.